src/classes/ERR_ExceptionHandler.cls

```diff
@@ -56,11 +57,16 @@ public with sharing class ERR_ExceptionHandler {
     */
     public static void beautifyExceptionMessage(Exception ex) {
         String message = ex.getMessage();
+        String typeName = ex.getTypeName();
```

Comment:
Yes, it is identifying the type of the exception to display the exception message